### ISSUE TITLE
Replace Kessie

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/operations#406]
Kessie's disks are 9 years old and the CPUs are 10 years old. We're starting to get failures in multiple drives.

We should replace the machine. Questions

1. Should the replacement go into Exonetric or elsewhere?
2. Is there something we want to put into Exonetric instead?

**Comment by Firefishy** [comment]
Yes, we should replace the server. The hosting is very good value. ;-)

**Comment by Firefishy** [comment]
Briefly discussed upgrade with Exonetric. They would be happy for us to replace the machine with something more modern. 1U is likely best.

**Comment by grischard** [comment]
Can we move the imagery stuff to Faffy instead?

**Comment by tomhughes** [comment]
Putting it on the dev server does not sound like a good idea to me for many, many reasons.

**Comment by pnorman** [comment]
kessie is shut down
